CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Change Log
 
+## [8.2.0](https://github.com/BurdaMagazinOrg/thunder-distribution/tree/8.2.0) 2017-06-01
+[Full Changelog](https://github.com/BurdaMagazinOrg/thunder-distribution/compare/8.1.5...8.2.0)
+
+Version 2.0 ads new functionality and improved updating to thunder. We bumbed the major version to 2 because there are 
+incompatibilities in the deployment. We removed the dependency on npm and bower for downloading javascript libraries.
+To be able to install thunder with composer you will have to add 
+
+    {
+         "type": "composer",
+         "url": "https://asset-packagist.org"
+    }
+
+to your repositories section in your composer.json  and the installer-path section of the same file should look like this:
+
+    "installer-paths": {
```

Comment:
Before "installer-paths", `"installer-types": ["bower-asset"],` is required too.